proxy/envoy/testdata/ingress-route-2.yaml.golden

```diff
@@ -0,0 +1,13 @@
+destination: foo.default.svc.cluster.local
+match:
+  http_headers:
```

Comment:
They are identical except what they point to. I can change them into `ingress-route-world.yaml` and `ingress-route-foo.yaml`, I guess.